src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -201,3 +201,41 @@
     :name                "My savings"
     :address             "0x43c...98d"
     :networks            [{:name :ethereum :short :eth}]}])
+
+(def buy-tokens-list
+  [{:title             "Ramp"
```

Comment:
curious where do we actually get this info from? 
Is it a predefined list or something we should get from status go? 🤔

Reply:
It's from the designs

Reply:
Sure, I get that. I am wondering if this is a finalized list. If so we can move it out of temp folder. 
Additionally if this data is available from status-go then I can create a follow up issue for someone to get this data from the correct point. 
Can you check with design/desktop of where the source of this data is?

Reply:
@J-Son89 the data is stored in status-go. Created an issue https://github.com/status-im/status-mobile/issues/17663

Reply:
awesome, thanks @OmarBasem :D